src/uproot/source/chunk.py

```diff
@@ -240,7 +244,7 @@ class Chunk:
     _dtype = numpy.dtype(numpy.uint8)
 
     @classmethod
-    def wrap(cls, source, data, start=0):
+    def wrap(cls, source: Source, data: numpy.ndarray, start: int = 0):
```

Comment:
I think there might be cases in which `wrap` is used to wrap `bytes`. These `memoryview` objects are used equivalently.

```suggestion
    def wrap(cls, source: Source, data: numpy.ndarray | bytes, start: int = 0):
```